__init__.py

```diff
@@ -0,0 +1,28 @@
+import pGan_fSplice
+from maskgen.plugins import findPlugin
+import os
+
+"""JT code only used for plugin conversion"""
+def transform(img, source, target, **kwargs):
+	donor = kwargs['donor']
+	pGan_fSplice.loadModel(os.path.join(findPlugin('GAN_face_splice'),pGan_fSplice.PREDICTOR_PATH))
+	pGan_fSplice.splice_donor_recipient(source,donor,target)
+	return None, None
```

Comment:
return {'subject': 'face', 'purpose':'add'}, None
to override any changes to the other args.